src/constants/routing.ts

```diff
@@ -163,6 +169,19 @@ export const COMMON_BASES: ChainCurrencyList = {
     WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON_MUMBAI] as Token,
     WETH_POLYGON_MUMBAI,
   ],
+  [SupportedChainId.CELO]: [
+    nativeOnChain(SupportedChainId.POLYGON),
+    WETH_POLYGON,
```

Comment:
These should be

    CEUR_CELO,
    CUSD_CELO,
    PORTAL_ETH_CELO,
    PORTAL_USDC_CELO,
    CMC02_CELO,

Reply:
🫠